rocketc/static/js/src/rocketc.js

```diff
@@ -1,22 +1,19 @@
 /* Javascript for RocketChatXBlock. */
 function RocketChatXBlock(runtime, element) {
 
-    function updateCount(result) {
-        $(".count", element).text(result.count);
-    }
+    $(function ($) {
+        /* Here's where you'd do things on page load. */
+    });
 
-    var handlerUrl = runtime.handlerUrl(element, "increment_count");
+    var setDefaultChannel = runtime.handlerUrl(element, "set_default_channel");
```

Comment:
This is something related to the method I ask you about above? the one I don't know where is called?

Reply:
That is the way to get the method from python and use in js like a api rest